### PR TITLE
New ADR, one ALB for backend

### PR DESCRIPTION
**Pull request by rtrinque**
Due to AWS limitation we will use the same ALB for internal and
external traffic for backend.

### CODE REVIEW

File: doc/architecture/decisions/0039-use-only-one-alb-for-backend.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## Status
+
+Accepted
+
+## Context
+
+This is the initial infrastructure that was planned for backend in AWS:
+
+ +-----+                  +-----+
```
Comment (sengi) [inline doc/architecture/decisions/0039-use-only-one-alb-for-backend.md:13]: Needs either enclosing in triple-backticks or indenting by 4 spaces to preserve the ascii art :)

File: doc/architecture/decisions/0039-use-only-one-alb-for-backend.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+|                                  |
++----------------------------------+
+
+This way we only need one target group per app to be associated to the ASG, which is under the limit of 50.
+
+
+## Consequences
+
+Internal traffic will now go outside of our VPC and go through the public IP of the ALB.
+While that doesn't seem ideal, in practice there should be very little concern about this setup. Traffic should not leave the AWS network and is encrypted anyway.
```
Comment (sengi) [inline doc/architecture/decisions/0039-use-only-one-alb-for-backend.md:72]: Worth mentioning also that this traffic is currently going over the Internet between Carrenza and AWS, so this proposal is still much better than what we currently have.

**Comment by issyl0** [comment]
Sorry but you're going to need to rebase this on master after I changed CI. innocent

**Comment by MuriloDalRi** [comment]
Hi, @rtrinque is this still needed or should we close this PR?

File: doc/architecture/decisions/0039-use-only-one-alb-for-backend.md (hunk 1 above)
Comment (ChrisBAshton) [reply doc/architecture/decisions/0039-use-only-one-alb-for-backend.md:14]: Hi @sengi - I've fixed up the ASCII formatting 👍 

Can you confirm whether or not this proposal was implemented? If so, I suggest we go ahead and merge this ADR. If not, let's close it.

Comment (sengi) [reply doc/architecture/decisions/0039-use-only-one-alb-for-backend.md:14]: There's gotta be an achievement badge for review necromancy 😀👻

It doesn't look like this was implemented in the end. I'm seeing two load balancers `govuk-backend-public` and `blue-backend-internal` and they both see significant traffic according to the monitoring tab in AWS.

backend prod ALB traffic sparklines 2022-09-30-nq8

**Comment by ChrisBAshton** [comment]
Closing, on the basis that [this was never implemented](https://github.com/alphagov/govuk-aws/pull/1285#discussion_r984346338).

**Comment by sengi** [comment]
Thanks for cleaning up these old PRs!

**Pull request closed without merging**